lib/HexBuffer.ts

```diff
@@ -0,0 +1,77 @@
+const ieee754 = require('ieee754'),
+    IntN = require('intn'),
+    intToHex = function (intV, isShort) {
```

Comment:
Can you add types to the function inputs of `intToHex` and `charToHex` as you did for the `addString` under `HexBuffer`? That would add additional type safety.

Something like: `intToHex = function (intV: number, isShort: boolean): string { ... }`

Reply:
-#1